src/go/rpk/pkg/cli/cmd/wasm/template/package_json.go

```diff
@@ -23,7 +23,7 @@ const packageJson = `{
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@vectorizedio/wasm-api": "^1.0.0"
```

Comment:
this version is the same as the Redpanda version, please change it for `^21.5.5`

Reply:
Are you sure ? This is the version of the wasm api that we push to npm, currently its 1.0.0 right?

Reply:
@graphcareful no, is the same as the current `$git-tag` latest in release. this should be generated or smth.

Reply:
https://www.npmjs.com/package/@vectorizedio/wasm-api

@andresaristizabal and @graphcareful  - we should really push our `readme.md` w/ every change to npm too. so it looks better. currently, our npm looks a bit jenky.